src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -1,13 +1,20 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.data.DataProvider;
+import edu.kis.vh.nursery.data.IntArrayStack;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class RhymersJUnitTest {
-//all test are working properly
+
+    final int EMPTY_STACK_VALUE = 0;
+    final int STACK_CAPACITY = 11;

```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY should remain 12.